src/main/java/org/jitsi/jicofo/conference/colibri/ColibriConferenceImpl.java

```diff
@@ -125,6 +125,16 @@ public void setGID(String gid)
         conferenceState.setGID(gid);
     }
 
+    @Override
+    public void setRtcStatsEnabled(boolean rtcStatsEnabled) {
+        conferenceState.setRtcStatsEnabled(rtcStatsEnabled);
```

Comment:
Have you tested that this? The code is super hairy and hard to follow, but I think the requests go through `colibriBuilder`, which gets reset at certain times, and `reset()` doesn't re-read the new flags

Reply:
I just did and it doesn't work :-( I guess you nailed the problem.. I'll dig into it.